common/src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java

```diff
@@ -558,11 +550,18 @@ private static BlockEntry<FusionCasingBlock> createFusionCasing(IFusionCasingTyp
         ALL_FUSION_CASINGS.put(casingType, casingBlock);
         return casingBlock;
     }
+    public static final BlockEntry<FusionCasingBlock> FUSION_CASING_SUPERCONDUCTOR = createFusionCasing(FusionCasingBlock.CasingType.FUSION_COIL);
```

Comment:
*ahem* I presume this is supposed to use FusionCasingBlock.CasingType.SUPERCONDUCTING_COIL

Reply:
I'm not surprised, the fusion reactor blocks kept confusing me on which one they're using since they use weird conventions for their names.

Reply:
Nevermind I'm just dumb :P